File: src/Problem01.java
public class Problem01 {
    public static int[] twoSum(int[] nums, int target) {

        for (int i = 0; i < nums.length - 1; i++) {
            if (nums[i] + nums[i + 1] == target) {
                return new int[]{i, i + 1};
            }
        }
        return new int[0];
    }
}

File: src/dart/problem_01.dart
// leetcode problem 01 - Two Sum
// https://leetcode.com/problems/two-sum/

class Solution {
  List<int> twoSum(List<int> nums, int target) {
      List<int> result = [];

      for (int i = 0; i < nums.length; i++) {
          for (int j = i + 1; j < nums.length; j++) {
              if (nums[i] + nums[j] == target) {
                  result.add(i);
                  result.add(j);
                  return result;
              }
          }
      }

      return result;
  }
}

